design/spaces.go

```diff
@@ -171,3 +171,11 @@ var _ = a.Resource("space", func() {
 		a.Response(d.Forbidden, JSONAPIErrors)
 	})
 })
+
+// genericLinks defines generic relations links
+var genericLinksForSpace = a.Type("GenericLinksForSpace", func() {
+	a.Attribute("self", d.String)
+	a.Attribute("related", d.String)
+	a.Attribute("meta", a.HashOf(d.String, d.Any))
+	a.Attribute("workItemTypeList", d.String)
```

Comment:
Thanks @baijum . done in https://github.com/almighty/almighty-core/pull/1033/commits/23418d21351a83cfd8244ff36fd854bde0008f82
Ref: https://github.com/almighty/almighty-core/blob/master/design/workitems.go#L14